server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -44,6 +44,9 @@ object CromwellEntryPoint extends GracefulStopSupport {
 
   val gracefulShutdown = config.as[Boolean]("system.graceful-server-shutdown")
 
+  // 3 minute DNS TTL down from JVM default of infinite [BA-6454]
+  java.security.Security.setProperty("networkaddress.cache.ttl" , "180")
```

Comment:
Would setting this property in our `reference.conf` or command line also work? I _think_ that this is equivalent to `-Dnetworkaddress.cache.ttl=...` on the command line, which is equivalent to `networkaddress.cache.ttl=...` in a config file. (Another possibly appropriate place might be the cromwell env file in [firecloud-develop](https://github.com/broadinstitute/firecloud-develop/blob/dev/run-context/live/configs/cromwell/app.env.ctmpl))

If I am right (a big IF...), I'd probably advocate for that instead of hard-coding here. If someone tried to set this to something different on the command line, this would presumably overrule those changes and set it back to 180.

Reply:
> [Y]ou can't set `networkaddress.cache.ttl` or `networkaddress.cache.negative.ttl` as System Properties by using the -D flag or calling `System.setProperty` because these are not System properties - they are Security properties

https://stackoverflow.com/a/17219327/818054

Reply:
It looks like inclusion in the `java.security` file could be an option though (we don't currently have one)

Reply:
Fair enough! In that case, no further questions.